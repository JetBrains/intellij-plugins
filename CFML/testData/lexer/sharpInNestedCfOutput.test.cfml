<cfoutput>
<cfif>
<cfoutput>text</cfoutput>
</cfif>
<cfif>
#myVar#
</cfif>
</cfoutput>