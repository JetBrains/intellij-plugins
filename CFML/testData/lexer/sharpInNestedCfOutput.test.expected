OPENER ('<')
CF_TAG_NAME ('cfoutput')
R_ANGLEBRACKET ('>')
WHITE_SPACE ('\n')
OPENER ('<')
CF_TAG_NAME ('cfif')
R_ANGLEBRACKET ('>')
WHITE_SPACE ('\n')
OPENER ('<')
CF_TAG_NAME ('cfoutput')
R_ANGLEBRACKET ('>')
CFML_TEMPLATE_TEXT ('text')
LSLASH_ANGLEBRACKET ('</')
CF_TAG_NAME ('cfoutput')
CLOSER ('>')
WHITE_SPACE ('\n')
LSLASH_ANGLEBRACKET ('</')
CF_TAG_NAME ('cfif')
CLOSER ('>')
WHITE_SPACE ('\n')
OPENER ('<')
CF_TAG_NAME ('cfif')
R_ANGLEBRACKET ('>')
WHITE_SPACE ('\n')
START_EXPRESSION ('#')
IDENTIFIER ('myVar')
END_EXPRESSION ('#')
WHITE_SPACE ('\n')
LSLASH_ANGLEBRACKET ('</')
CF_TAG_NAME ('cfif')
CLOSER ('>')
WHITE_SPACE ('\n')
LSLASH_ANGLEBRACKET ('</')
CF_TAG_NAME ('cfoutput')
CLOSER ('>')